common-lib/src/main/scala/com/gu/mediaservice/lib/aws/DynamoDB.scala

```diff
@@ -61,6 +61,14 @@ class DynamoDB[T](config: CommonConfig, tableName: String, lastModifiedKey: Opti
     }
   }
 
+  def keySet(id: String, key: String, value: String)
```

Comment:
how does this differ from stringSet, and should this field be stored as a string?

Reply:
right `stringSet` does the same, updating the PR now

> I don't think we should add the checkbox there, we've managed to go a few weeks without a combined BBC/gu meeting. @pradasa is looking at this checkbox area for us to do something similar with

okay I will remove it for now